editor/plugins/curve_editor_plugin.h

```diff
@@ -36,37 +36,34 @@
 #include "editor/editor_resource_preview.h"
 #include "scene/resources/curve.h"
 
+class EditorSpinSlider;
+class MenuButton;
 class PopupMenu;
 
-// Edits a y(x) curve
-class CurveEditor : public Control {
-	GDCLASS(CurveEditor, Control);
+class CurveEdit : public Control {
```

Comment:
It is a convention Geometror suggested:

> The name of the CurveEditorRect isn't really descriptive, maybe CurveEdit (arguably only slightly better, because it's a bit more consistent with Godot's naming and therefore maybe better to understand)

I haven't even updated the PR description to the new name. Anyway, I don't know about the convention, but I'd keep it for consistency for now.

Reply:
I'm still not the greatest fan of this since the two names are so similar - so at least putting a couple of comments in the .h file explaining what each is would be nice. This way if someone is confused, like I am all the time 🙃, they can always go to the definition of each class and be reminded of what it is!